moveit_ros/planning/rdf_loader/src/rdf_loader.cpp

```diff
@@ -38,9 +38,11 @@
 #include <moveit/rdf_loader/rdf_loader.h>
 #include <moveit/profiler/profiler.h>
 
-// ROS
-#include <ros/ros.h>
-#include <ros/package.h>
+// ROS 2
+#include <rclcpp/rclcpp.hpp>
+#include <std_msgs/msg/string.hpp>
+#include <ament_index_cpp/get_package_prefix.hpp>
```

Comment:
Is this needed?

Reply:
I included it because I'm using the `PackageNotFoundError` exception in this [line](https://github.com/JafarAbdi/moveit2/blob/pr-rdf_loader/moveit_ros/planning/rdf_loader/src/rdf_loader.cpp#L248) and it's inside the included header file [see](https://github.com/ament/ament_index/blob/master/ament_index_cpp/include/ament_index_cpp/get_package_prefix.hpp#L27)